app/forms/personal_details_form.rb

```diff
@@ -132,15 +130,15 @@ def other_nationalities_hash
   end
 
   def date_of_birth_valid
-    errors.add(:date_of_birth, :invalid) unless date_of_birth.is_a?(Date)
-  end
-
-  def date_of_birth_not_in_future
-    errors.add(:date_of_birth, :future) if date_of_birth.is_a?(Date) && date_of_birth > Time.zone.today
-  end
-
-  def date_of_birth_year_is_four_digits
-    errors.add(:date_of_birth, :invalid_year) if date_of_birth.is_a?(Date) && date_of_birth.year.digits.length != 4
+    if !date_of_birth.is_a?(Date)
```

Comment:
`validate :date_of_birth_valid` sounds a bit 🤮 . I think the intention was clearer with the separate methods but I realise checking that's it's a date everywhere is also a bit annoying.

Reply:
Aw man, happy face from John, sad face from G :(

The elsif flow is more consistent with the rest of our validations e.g. `commencement_date_valid` too (although I know that's not necessarily a reason to continue doing something)

I think I'll leave as is now.